src/components/layout/nav/PublicMenu.tsx

```diff
@@ -43,8 +43,7 @@ export default function PublicMenu() {
   const { t } = useTranslation()
   const { data: session } = useSession()
   const [anchorEl, setAnchorEl] = useState<Element | null>(null)
-  const open = Boolean(anchorEl)
-
+  const defaultAvatar = '/podkrepi-bg-logo-avatar-black.svg'
```

Comment:
About that black file. Is that what design has provided. I personally don't like that black color of the logo and feel like a darker gray would be more suitable, something like `#c7c7c7`

Reply:
I removed it in my second commit. We can try to improve it to sit better and add it back in the future

Reply:
Resolving it then. You can contact design about that if you think they might reiterate